tjpcov/covariance_calculator.py

```diff
@@ -147,29 +143,30 @@ def get_covariance_terms(self):
         return self.cov_terms
 
     def create_sacc_cov(self, output="cls_cov.fits", save_terms=True):
-        """
-        Write the sacc file with the total covariance.
-
-        Parameters
-        ----------
-            output (str): Filename. This will be joined to the outdir path
-            specified in the configuration file.
-            save_terms (bool): If true, save individual files for each of the
-            requested contributions. The will have the covariance term (e.g.
-            gauss) appended to the filename (before the extension, e.g.
-            cls_cov_gauss.fits)
+        """Write the sacc file with the total covariance.
+
+        Args:
+            output (str, optional): Filename. This will be joined to the outdir
+                path specified in the configuration file. Default
+                "cls_cov.fits"
+            save_terms (bool, optional): If true, save individual files for
+                each of the requested contributions. The will have the
+                covariance term (e.g. gauss) appended to the filename (before
+                the extension, e.g. cls_cov_gauss.fits)
```

Comment:
Maybe add a Returns: part to the docstring now that we are returning?

Reply:
Done